### PR TITLE
22 moving getstarted readme to wiki

### LINKED ISSUE #22: Moving getstarted readme to wiki
With PR [#358](https://github.com/oakestra/oakestra/pull/358#discussion_r1850561782) we're remvoing the getstarted guide form the readme and we keep it centralized here.

### PR DESCRIPTION
**Pull request by giobart**
fixes #22

While the PR looks quite bit, the changes are not many. 

1: Ported the github oakestra/README.md in this wiki inside the **Your first oak cluster** and **deploy your fisrt app** sections.

2: Then I added a wiki archive sction for the older versions of the wiki: 
image

Using the releases button it is now possible to navigate older version of the same wiki 

When navigating a version of the wiki which is not the latest, a banner will tell the user to refer to the latest version. 
image

### CODE REVIEW

**Review by nitindermohan — APPROVED** [review]
Approved!

**Comment by Malyuk-A** [comment]
At least on my laptop - when I open the landing page the logo in the middle is cut off a bit from the top
image

**Comment by Malyuk-A** [comment]
I think this page is quite outdated - be it missing new people - or the legacy nitinder tum email
image

**Comment by Malyuk-A** [comment]
BTW - my finding could be not relevant because we are replacing this docs with the new ones anyways - so feel free to ignore

File: config.toml [hunk 1]
```diff
@@ -87,11 +87,11 @@ archived_version = false
 # The version number for the version of the docs represented in this doc set.
 # Used in the "version-banner" partial to display a version number for the 
 # current doc set.
-version = "0.0"
+version = "0.4.301"
 
 # A link to latest version of the docs. Used in the "version-banner" partial to
 # point people to the main doc site.
-#url_latest_version = "https://example.com"
+url_latest_version = "https://example.com"
```
Comment (Malyuk-A) [inline config.toml:94]: It seems a bit strange to have this uncommented

**Review by Malyuk-A — APPROVED** [review]
Hmm as mentioned before - because this will be replaced soon anyway - this seems to be fine to me. I tested it locally and it works.

**Comment by Malyuk-A** [comment]
Also fun - I lack write access to this repo so my approval does not count to get the 2/2 😅
image

**Comment by giobart** [comment]
> Also fun - I lack write access to this repo so my approval does not count to get the 2/2 😅
Solved :D

**Pull request merged**